src/Auxiliary/Aux_ComputeProfile.cpp

```diff
@@ -266,9 +388,15 @@ void Aux_ComputeProfile( Profile_t *Prof[], const double Center[], const double
 #                          ifdef GRAVITY
                            case _POTE:
                            {
-                              const real Weight = FluidPtr[DENS][k][j][i]*dv;    // weighted by cell mass
-
-                              OMP_Data  [p][TID][bin] += PotPtr[k][j][i]*Weight;
+                              const real Weight = ( FluIntTime )    // weighted by cell mass
```

Comment:
Just a reminder. Make sure that all fluid interpolation parameters are properly set even when only preparing `_POTE`.